client/me/purchases/purchase-item/index.jsx

```diff
@@ -55,65 +49,95 @@ class PurchaseItem extends Component {
 		);
 	}
 
-	renewsOrExpiresOn() {
+	getStatus() {
 		const { purchase, translate, moment } = this.props;
+		const expiry = moment( purchase.expiryDate );
 
-		if ( showCreditCardExpiringWarning( purchase ) ) {
-			return (
-				<Notice isCompact status="is-error" icon="notice">
-					{ translate( 'Credit card expiring soon' ) }
-					{ this.trackImpression( 'credit-card-expiring' ) }
-				</Notice>
-			);
+		if ( purchase && isPartnerPurchase( purchase ) ) {
+			return translate( 'Managed by %(partnerName)s', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'This plan is managed by %(partnerName)s.' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/purchase-listing-view)